components/org.wso2.carbon.identity.user.endpoint/src/test/java/org/wso2/carbon/identity/user/endpoint/impl/MeApiServiceImplTest.java

```diff
@@ -63,20 +64,44 @@
     private MeApiServiceImpl meApiService;
 
     @Test
-    public void testResendCodePost() throws Exception {
-
-        mockStatic(IdentityUtil.class);
-        mockStatic(Utils.class);
-        PowerMockito.when(IdentityUtil.class, (String) IdentityUtil.threadLocalProperties.get().
-                get(Constants.TENANT_NAME_FROM_CONTEXT)).thenReturn("TestTenant");
-        when(Utils.getUserSelfRegistrationManager()).thenReturn(userSelfRegistrationManager);
+    public void testMePost() throws Exception {
```

Comment:
missing method comments.

Reply:
As this is a test method do we really need to add method comments. If so we need to add method comments to all the test methods as we have not added them :)